README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
The `entry` command is synonymous to `entries` but does something entirely different. That's odd and should therefore probably read as `addentryitem` as that reflects more what this command does.

@mtdowling has a good point though that people rather use an editor to modify the changelog and that's also why I thought about adding a command `edit` that opens the systems default editor with the content of the latest entry and replaces the content of that entry with the edits made in that editor session. It would behave much alike `git commit` does.

Further the commands are not intuitive to me. The following is a summary of all suggestions:

* `create` could be `init` (alike `git init`)
* `entries` could be `list` or `showentries` or `listitems`
* `next` could be `add` or `addentry` or `additem` or `addnext` or `addunreleased`
* add new command `edit` or `editentry` or `edititem` or `editchanges`
* `contents` could be `show` or `showentry` or `showitem` or `showchanges`
* `update` could be `setversion` or `setentryversion` or `setitemversion`
* `entry` could be `addentryitem` or `addchange`

The first item is what I see as the best choice.